README.md

```diff
@@ -42,6 +42,19 @@ For async communication and long running discussions please use issues and pull
 
 For sync communication we have an IRC channel #cri-o, on chat.freenode.net, that everyone is welcome to join and chat about development.
 
+## Compatibility matrix: CRI-O <-> Kubernetes clusters
+
+| Version - Branch        | Kubernetes branch/version     | Maintenance status |
+|-------------------------|-------------------------------|--------------------|
+| CRI-O 1.0  - kube-1.6.x | Kubernetes 1.6 branch, v1.6.x | =                  |
+| CRI-O 1.1  - kube-1.7.x | Kubernetes 1.7 branch, v1.7.x | ✓                  |
+| CRI-O HEAD - master     | Kubernetes master branch      | ✓                  |
+
+Key:
+
+* `✓` Changes in main Kubernetes repo about CRI are actively implemented in CRI-O
```

Comment:
The CRI changes being tracked and implemented is the most important part, but we obviously also are going to keep the overall compatibility with Kubernetes. May be this should be explicitly called out.